pkgs/development/python-modules/simpervisor/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pytest
+, pytest-asyncio
+, aiohttp
+}:
+
+buildPythonPackage rec {
+  pname = "simpervisor";
+  version = "0.4";
+
+  src = fetchFromGitHub {
+    owner = "yuvipanda";
+    repo = "simpervisor";
+    rev = "v${version}";
+    sha256 = "1brsisx7saf4ic0dih1n5y7rbdbwn1ywv9pl32bch3061r46prvv";
+  };
+
+  checkInputs = [
+    pytest
+    pytest-asyncio
+    aiohttp
+  ];
+
+  checkPhase = ''
```

Comment:
Please use pytestCheckHook and disabledTests.

Reply:
`disabledTestFiles`, not `disabledTests`.

Reply:
And now actually `disabledTestPaths`.